ci-operator/jobs/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master-presubmits.yaml

```diff
@@ -291,6 +291,7 @@ presubmits:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-sandboxed-containers-operator-master-sandboxed-containers-operator-e2e
+    optional: true
```

Comment:
Nit: change commit title to "sandboxed-containers-operator: make the e2e job optional" so that it fits on one line?